modules/migration-mtc-release-notes-1-7-15.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-7-15_{context}"]
+= {mtc-full} 1.7.15 release notes
+
+[id="resolved-issues-1-7-15_{context}"]
+== Resolved issues
+
+This release has the following resolved issues:
+
+.CVE-2024-24786: A flaw was found in Golang's protobuf module, where the unmarshal function can enter an infinite loop
+
+A flaw was found in the `protojson.Unmarshal` function, which could enter an infinite loop when unmarshaling certain forms of invalid JSON. This condition could occur when unmarshaling into a message that contained a `google.protobuf.Any` value or when the `UnmarshalOptions.DiscardUnknown` option was set.
+
+To resolve this issue, upgrade to {mtc-short} 1.7.15.
+
+For more details, see link:https://access.redhat.com/security/cve/CVE-2024-24786[(CVE-2024-24786)].
+
+.CVE-2024-28180: `jose-go` improper handling of highly compressed data
+
+A vulnerability was found in Jose due to improper handling of highly compressed data. An attacker could send a JSON Web Encryption (JWE) containing compressed data that used large amounts of memory and CPU when decompressed by Decrypt or DecryptMulti. 
+
+To resolve this issue, upgrade to {mtc-short} 1.7.15.
+
+For more details, see link:https://access.redhat.com/security/cve/CVE-2024-28180[(CVE-2024-28180)].
+
+
+[id="known-issues-1-7-15_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Direct Volume Migration is failing as the rsync pod on the source cluster goes into an `Error` state
+
+On migrating any application with Persistent Volume Claim (PVC), Stage succeeds with warnings and Direct Volume Migration (DVM) fails with the `rsync` pod on the source namespace going into an `error` state. link:https://bugzilla.redhat.com/show_bug.cgi?id=2256141[(BZ#2256141)]
```

Comment:
"On migrating any application with Persistent Volume Claim (PVC)"

Not sure what is meant here:

> An application that contains a PVC or migrating an application by using a PVC.

Reply:
"Stage" is a migration operation? Might be worth stating this so a supporting noun is evident.